src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseEventFieldDefinition.java

```diff
@@ -161,4 +162,13 @@ public void setPublishAs(String publishAs) {
         this.publishAs = publishAs;
     }
 
+    @ApiModelProperty(value = "")
```

Comment:
Let's add a basic description here for Swagger (example suggestion below, feel free to word better!)
```suggestion
    @ApiModelProperty(value = "Default value for the case field, if no existing value")
```